src/RWS.jl

```diff
@@ -128,3 +130,115 @@ function Init(rws::RewritingSystem = RewritingSystem(),
 
     return rws
 end
+
+"""
+    BuildRWS(G::FPGroup)
+
+Builds a RewritingSystem based on a finitely-presented group.
+This function is based on the ./deps/src/kbmag-1.5.8/standalone/lib/rwsio.c:224-525 implementation.
+"""
+function BuildRWS(G::Groups.FPGroup;
+                  isConfluent = false,
+                  tidyint = 0,
+                  maxeqns = 32767,
+                  maxstates = 0,
+                  confnum = 500,
+                  sorteqns = false,
+                  maxoplen = 0,
+                  maxlenleft = 0,
+                  maxlenright = 0,
+                  maxoverlaplen = 0,
+                  maxwdiffs = 512,
+                  maxreducelen = 32767,
+                  rkminlen = 0,
+                  rkmineqns = 0,
+                  ordering::KBMOrderings = SHORTLEX)
+    rws = Init()
+
+    # Do not set the name
+    rws.name = ntuple(_->Cchar(0), 256)
+
```

Comment:
just `rws = RewritingSystem()` is enough

Reply:
Yes, but it might be a good idea to Init() the system. I am not sure whether the initial values are used by the KBmag library in any way. It is a constant-time operation.